iidm/iidm-api/src/main/java/com/powsybl/iidm/network/ThreeWindingsTransformer.java

```diff
@@ -302,6 +303,10 @@ default Leg setRatedS(double ratedS) {
      */
     Substation getSubstation();
 
+    default Optional<Substation> getOptionalSubstation() {
```

Comment:
The naming is really ugly and we are going ti use it EVERYWHERE. Even if it will create a breaking change, should we consider just replacing the getSubstation ?

Reply:
Yes, I think you're right